src/mlagility/analysis/util.py

```diff
@@ -1,16 +1,12 @@
-import os
```

Comment:
How do you feel about making `-d CACHE_DIR` a positional argument for both `delete` and `clean`?

Reply:
Discussed offline, we will keep `-d`